charts/prometheus-systemd-exporter/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: prometheus-systemd-exporter
 description: A Helm chart for prometheus systemd-exporter
 type: application
-version: 0.3.0
-appVersion: "0.6.0"
+version: 1.0.0
+appVersion: "0.7.0"
```

Comment:
could you do a separate PR for this so that we can revert the change if its a bugged version